src/Persistence/MassTransit.EntityFrameworkCoreIntegration/MassTransit.EntityFrameworkCoreIntegration.csproj

```diff
@@ -14,7 +14,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.6" />
+      <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.0" />
```

Comment:
Lets remove the explicit upgrade of Microsoft.Data.SqlClient, and instead let it resolve through the transitive dependency within Microsoft.EntityframeworkCore.  I always prefer to use the minimum, and let the user of the library upgrade if they want.